MeshLib/MeshEditing/DuplicateMeshComponents.cpp

```diff
@@ -19,8 +19,8 @@
 
 namespace MeshLib
 {
-
-std::vector<MeshLib::Node*> copyNodeVector(const std::vector<MeshLib::Node*> &nodes)
```

Comment:
Push the clang-format changes elsewhere (like master). They are not part of the tooling.